test/SpokeUtils.ts

```diff
@@ -24,8 +54,20 @@ describe("SpokeUtils", function () {
     fillDeadline: random(),
     exclusiveRelayer: randomAddress(),
     exclusivityDeadline: random(),
+    ...dummyFillProps,
   };
 
+  let spokePool: Contract;
+  let deploymentBlock: number;
+
+  beforeEach(async function () {
+    const { spokePool: _spokePool, deploymentBlock: _deploymentBlock } = await deploySpokePoolWithToken(
+      sampleData.originChainId
+    );
+    spokePool = _spokePool;
+    deploymentBlock = _deploymentBlock;
+  });
```

Comment:
Minor - the reassignment to _ prefix seems unnecessary here, since both of the assigned variables are rw anyway. Does this work?

```suggestion
  let spokePool: Contract;
  let deploymentBlock: number;

  beforeEach(async function () {
    ({ spokePool, deploymentBlock } = await deploySpokePoolWithToken(sampleData.originChainId));
  });
```